SU-SWS/saa_victoria_subtheme
# Adapt 443  fat footer

# READY FOR REVIEW

# Summary
- Adjusted link labels, padding, fixed saa logo, tweaked legal links width

# Review By (Date)
- 8/14

# Urgency
- medium

# Steps to Test

1. Pull down the branch
2. Clear the site cache
3. check that all 6 link sections align above each other correct
4. check that saa footer logo has a black | not red
5. check that legal links breaks to 2 lines before Terms of Use on large desktop
6. link sections like Clubs and groups and others use a & not and

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
